src/PhpSpreadsheet/Calculation/Logical/Operations.php

```diff
@@ -116,22 +85,7 @@ public static function logicalOr(...$args)
      */
     public static function logicalXor(...$args)
     {
-        $args = Functions::flattenArray($args);
-
-        if (count($args) == 0) {
-            return ExcelError::VALUE();
-        }
-
-        $args = array_filter($args, function ($value) {
-            return $value !== null || (is_string($value) && trim($value) == '');
-        });
-
-        $returnValue = self::countTrueValues($args);
-        if (is_string($returnValue)) {
-            return $returnValue;
-        }
-
-        return $returnValue % 2 == 1;
+        return self::countTrueValues($args, 'xor');
```

Comment:
I might be tempted to have three wrapper methods, `countTrueValuesForAnd()`, `countTrueValuesForOr()`, `countTrueValuesForXor()` that call the basic `countTrueValues()` method, then handle the specific 0 count and comparison logic, rather than passing a string to the one method and then using if tests

Reply:
I changed to eliminate the ifs, not precisely the way you wanted, but addressing your concern, I hope.